Base/Python/slicer/util.py

```diff
@@ -140,29 +140,20 @@ def findChildren(widget=None,name="",text="",title="",className=""):
     if not hasattr(p,'children'):
       continue
     parents += p.children()
-    if name and fnmatch.fnmatchcase(p.name, name):
-      children.append(p)
-    elif text:
-      try:
-        p.text
-        if fnmatch.fnmatchcase(p.text, text):
-          children.append(p)
-      except (AttributeError, TypeError):
-        pass
-    elif title:
-      try:
-        p.title
-        if fnmatch.fnmatchcase(p.title, title):
-          children.append(p)
-      except AttributeError:
-        pass
-    elif className:
-      try:
-        p.className()
-        if fnmatch.fnmatchcase(p.className(), className):
-          children.append(p)
-      except AttributeError:
-        pass
+    condition = True
```

Comment:
Rename this variable to something more specific, which makes it clear what True and False mean. For example matched_filter_criteria would be specific enough.